### ISSUE TITLE
Notice regarding PxView on Google Play

### ISSUE THREAD

**Issue by alphasp** [alphasp/pxview#66]
PxView is currently suspended and removed by Google, hence it not available on Google Play now.
In the mean, the app apk can be download from https://github.com/alphasp/pxview/releases/tag/v4.7.0 .

I am still trying to appeal to Google to restore the app, if my appeal is not granted, my only options will be either submit as a new app with different id & name or only available within github.

**Comment by bb010g** [comment]
Would [IzzyOnDroid](https://android.izzysoft.de/repo) ([info](https://android.izzysoft.de/repo/info)) / [F-Droid](https://f-droid.org/) ([info](https://f-droid.org/en/docs/FAQ_-_App_Developers/)) be considered, to make updates easier?

(I also want to note that even if you use F-Droid in the future, as long as you [submit your signature](https://f-droid.org/en/docs/Reproducible_Builds/), users will be able to smoothly transition between your builds (distributed on GitHub) & F-Droid's builds.)

**Comment by paul12sava** [comment]
Please, could you upload an ipa file for ios ?

**Comment by alphasp** [comment]
> Please, could you upload an ipa file for ios ?

I don't think this is possible without paid Apple developeer account and without limit

**Comment by bb010g** [comment]
I [submitted PxView to IzzyOnDroid](https://gitlab.com/IzzyOnDroid/repo/-/issues/153), but got this response:

> Thanks for the suggestion! But the APK is beyond the 30M limit unfortunately. The app further comes with at least Crashlytics, Firebase and GMS. This is a quite unfortunate combination, so I'm afraid I must decline, sorry.

**Comment by nagaisa** [comment]
was thare a resion sayed why it was removed? but yea as bb010g sayed f-driod might be a good idea

**Comment by Baker-ux-dot** [comment]
Can you make a proprietary version that includes Crashlytics and an F-Droid version that does not include it?
I would like to be able to switch between them in the build options.

**Comment by alphasp** [comment]
Sorry for the long wait, PxView is now available on Google Play again https://play.google.com/store/apps/details?id=com.utopia.pxviewr under new name PxView R.

**Comment by Jacinto98** [comment]
I cannot seem to find the app on Google play, region locked ? Can you upload APK?

**Comment by alphasp** [comment]
> I cannot seem to find the app on Google play, region locked ? Can you upload APK?

PxView is currently suspended and removed by Google, hence it not available on Google Play now.
In the mean, the app apk can be download from https://github.com/alphasp/pxview/releases/tag/v5.0.0

I am busy recently but I will be working on new version that will be comply with Google Play policy. Stay tuned.

**Comment by bb010g** [comment]
@alphasp What were the Play Store's issues with PxView R?

**Comment by alphasp** [comment]
> @alphasp What were the Play Store's issues with PxView R?

PxView R app was suspended due to the app contains content that doesn't comply with Google Play Sexual Content and Profanity policy. 
It is mainly due to the app not handling sanity level of the work, ie: some works are not R-18 but still deem as over sexual under Google Play policy. Official pixiv app do handle this correctly as the app will check for each work sanity level and not showing those work by default in app.

**Comment by gwpl** [comment]
Will this censorship mechanism be not available on e.g. F-Droid version, therefore also work to update/install from there?

**Comment by dic-l-art** [comment]
Will this censorship mechanism be not available on e.g. F-Droid version, therefore also work to update/install from there?

**Comment by dic-l-art** [comment]
Do I understand correctly that if I use playstore version, then censorship is applied (that's why "R" in name),
while if I use new apt ! version 5.0 currently) then it will work without censorship?
